source/agora/consensus/data/Transaction.d

```diff
@@ -169,6 +166,14 @@ unittest
     assert(!txs.isStrictlyMonotonic!((a, b) => identity(a) < identity(b)));
 }
 
+/// Indicates if output is frozen (staking), payment (spending) or coinbase (rewards / fees)
+public enum OutputType : ubyte
+{
+    Payment,
+    Freeze,
+    Coinbase,
```

Comment:
Can we find the refund output?

Reply:
When creating a transaction with one or more `freeze` outputs, the refund output will be a single `output` of type `Payment`.